BuildProject/App.tsx

```diff
@@ -0,0 +1,139 @@
+            };
+
+            PromotedMetrics.startSessionAndLogUser("foobar");
+            logPassed('startSessionAndLogUser');
+
+            PromotedMetrics.startSessionAndLogSignedOutUser();
+            logPassed('startSessionAndLogSignedOutUser');
+
+            PromotedMetrics.logImpression(content);
+            logPassed('logImpression');
+
+            PromotedMetrics.logViewReady("foobar", "batman");
+            logPassed('logViewReady');
+
+            PromotedMetrics.logViewChange("spaghetti", "meatballs");
+            logPassed('logViewChange');
+
+            PromotedMetrics.collectionViewDidLoad("hello");
+            logPassed('collectionViewDidLoad');
+
+            PromotedMetrics.collectionViewDidChange([], "hello");
+            logPassed('collectionViewDidChange');
+
+            PromotedMetrics.collectionViewDidUnmount("hello");
+            logPassed('collectionViewDidUnmount');
+
+            PromotedMetrics.getLoggingSessionInfo();
+            logPassed('getLoggingSessionInfo');
+
+            PromotedMetrics.logNavigateAction("screen");
+            logPassed('logNavigateAction');
+
+            PromotedMetrics.logNavigateActionWithContent("screen", content);
+            logPassed('logNavigateActionWithContent');
+
+            PromotedMetrics.logAddToCartAction(content);
+            logPassed('logAddToCartAction');
+
+            PromotedMetrics.logRemoveFromCartAction(content);
+            logPassed('logRemoveFromCartAction');
+
+            PromotedMetrics.logCheckoutAction();
+            logPassed('logCheckoutAction');
+
+            PromotedMetrics.logPurchaseAction(content);
+            logPassed('logPurchaseAction');
+
+            PromotedMetrics.logShareAction(content);
+            logPassed('logShareAction');
+
+            PromotedMetrics.logLikeAction(content);
+            logPassed('logLikeAction');
+
+            PromotedMetrics.logUnlikeAction(content);
+            logPassed('logUnlikeAction');
+
+            PromotedMetrics.logCommentAction(content);
+            logPassed('logCommentAction');
+
+            PromotedMetrics.logMakeOfferAction(content);
+            logPassed('logMakeOfferAction');
+
+            PromotedMetrics.logAskQuestionAction(content);
+            logPassed('logAskQuestionAction');
+
+            PromotedMetrics.logAnswerQuestionAction(content);
+            logPassed('logAnswerQuestionAction');
+
+            PromotedMetrics.logCompleteSignInAction();
+            logPassed('logCompleteSignInAction');
+
+            PromotedMetrics.logCompleteSignUpAction();
+            logPassed('logCompleteSignUpAction');
+
+            PromotedMetrics.logAction("custom");
+            logPassed('logAction');
+
+            PromotedMetrics.logActionWithType("custom", ActionType.Navigate);
+            logPassed('logActionWithType');
+
+            PromotedMetrics.logActionWithContent("custom", ActionType.Share, content);
+            logPassed('logActionWithContent');
```

Comment:
Can you update the library code with comments that indicate that this file should be updated? 
 I don't have a great way to keep these in sync.  I think a comment is okay for now.

Reply:
Done.